src/group-tabs.vue

```diff
@@ -80,6 +46,23 @@ watch(
 	}
 );
 
+function isEqual(obj1: any, obj2: any) {
```

Comment:
I prefer to keep using isEqual from lodash, no need to implement it ourselves. Right now we implicitly depend on the lodash that is bundled with directus/app, it is probably a good idea to add `lodash-es` to the dev dependencies. They bundle should still stay small.